frontend/src/lib/experimental/FeedbackCallCTA.tsx

```diff
@@ -74,7 +74,7 @@ const feedbackCallLogic = kea<feedbackCallLogicType>({
     events: ({ actions, values, cache }) => ({
         afterMount: () => {
             if (
-                values.featureFlagGroup?.startsWith('variant') &&
+                values.featureFlagGroup?.startsWith?.('variant') &&
```

Comment:
flyby bug fix that kept breaking dev env

Reply:
🙏 I was just hitting the same thing. I think it was the `sync_feature_flag` command setting a feature flag to false that was supposed to be a mutli-variant.

Reply:
oh, interesting, makes sense.